ci/build_setup.sh

```diff
@@ -100,6 +100,7 @@ BAZEL_BUILD_OPTIONS=(
   "--repository_cache=${BUILD_DIR}/repository_cache"
   "--experimental_repository_cache_hardlinks"
   "--action_env=CLANG_FORMAT"
+  "--jobs=${NUM_CPUS}"
```

Comment:
i think this is already set in `ci/setup_cache.sh` -  it is set to `--jobs=HOST_CPUS*.9`

Reply:
@phlax Thanks a lot for the review!

`--jobs=HOST_CPUS*.9` is set only in case `BAZEL_REMOTE_CACHE` is set too.
I am building locally without remote cache. Is this scenario rare and I am optimizing a non-interesting use case?

As to the PR itself, I can refactor such that both cases are handled similar to the way it is handled in `setup_cache.sh`, by introducing a new env var `NUM_JOBS` with default value`HOST_CPUS*.9` (for example). This way it will be possible to override, and will be handled in a unified way with or without remote cache. Does it sound good?

Reply:
I am running it like this `./ci/run_envoy_docker.sh './ci/do_ci.sh bazel.dev'`.
This takes 8/8 cores. I would like to limit the whole thing to 2 cores at most.

Reply:
I am running it like this `./ci/run_envoy_docker.sh './ci/do_ci.sh bazel.dev'`.
This takes 8/8 cores. I would like to limit the whole thing to 2 cores at most.

Reply:
@phlax OK. Thanks!
Dropping the PR ..